src/app/pages/give/give.js

```diff
@@ -202,6 +214,12 @@ export default class Give extends CMSPageController {
         if (invalid) {
             event.preventDefault();
             this.update();
+        } else {
+            localStorage.setItem('donation:first_name', this.el.querySelector('[name=First_Name]').value);
```

Comment:
These names don't match the form elements, capitalization-wise, and you're using `AMT` for `Donation_Amount__c`